application/actions/meetings_test.go

```diff
@@ -61,3 +62,76 @@ func (as *ActionSuite) Test_MeetingsList() {
 	as.NotContains(body, mtgs[1].Name, "should not have included name of recent meeting")
 
 }
+
+func (as *ActionSuite) Test_meetingsJoin() {
+	f := createFixturesForMeetings(as)
+
+	type testCase struct {
+		name       string
+		inviteCode string
+		location   models.Location
+		meeting    models.Meeting
+		user       models.User
+		wantInvite models.MeetingInvite
+	}
+
+	testCases := []testCase{
+		{
+			name:     "already a participant",
+			location: f.Locations[2],
+			meeting:  f.Meetings[2],
+			user:     f.Users[2],
+		},
+		{
+			name:     "meeting creator",
+			location: f.Locations[0],
+			meeting:  f.Meetings[0],
+			user:     f.Users[0],
+		},
+		{
+			name:       "regular user",
+			inviteCode: f.MeetingInvites[3].Secret.String(),
+			location:   f.Locations[3],
+			meeting:    f.Meetings[3],
+			user:       f.Users[1],
+			wantInvite: f.MeetingInvites[3],
+		},
+	}
+
+	for _, tc := range testCases {
+		as.T().Run(tc.name, func(t *testing.T) {
+
+			reqBody := api.MeetingParticipantInput{
+				MeetingID: tc.meeting.UUID.String(),
+			}
+			if tc.inviteCode != "" {
+				reqBody.Code = &tc.inviteCode
+			}
+
+			req := as.JSON("/events")
+			req.Headers["Authorization"] = fmt.Sprintf("Bearer %s", tc.user.Nickname)
+			req.Headers["content-type"] = "application/json"
+			res := req.Post(reqBody)
+
+			body := res.Body.String()
+			as.Equal(200, res.Code, "incorrect status code returned, body: %s", body)
+
+			wantContains := []string{
+				fmt.Sprintf(`"nickname":"%s"`, tc.user.Nickname),
```

Comment:
```suggestion
				fmt.Sprintf(`"created_by":{"id":"%s"`, tc.user.UUID),
```
to make sure it's actually checking the creator and not just any participant

Reply:
:+1: